### ISSUE TITLE
Cannot read property 'firstCreatePass' of null - After update to angular 10

### ISSUE THREAD

**Issue by suhailkc** [rsaenen/ngx-custom-validators#40]
I have got this error after an update from Angular 9 to Angular 10.

**HTML:**
```
<input class="form-control" type="text" placeholder="digits" ngModel name="digits" #digits="ngModel" digits/>
```

**Error in console:**
```
core.js:4073 ERROR TypeError: Cannot read property 'firstCreatePass' of null
    at providersResolver (core.js:27050)
    at Object.definition.providersResolver (core.js:27382)
    at resolveDirectives (core.js:7918)
    at elementStartFirstCreatePass (core.js:13866)
    at ɵɵelementStart (core.js:13904)
    at ɵɵelement (core.js:13979)
    at FormValidationComponent_Template (template.html:47)
    at executeTemplate (core.js:7364)
    at renderView (core.js:7173)
    at renderComponent (core.js:8431)
```

  - OS: macOS Catalina
  - Angular Version: 10.0.1
  - Typescript version: 3.9.5
  - npm version: 6.13.4
  - Node Version: 12.14.0

**Comment by bt-serdarbuyuktemiz** [comment]
downgrade ngx-custom-validators to version 8

**Comment by sannyjacobsson** [comment]
I have the same issue. Any information to when, or how (without downgrading), this issue can be resolved?

**Comment by matiasah** [comment]
Does anyone have a solution for this?

**Comment by rsaenen** [comment]
Released with 10.0.0, please reopen the issue if necessary.

**Comment by dtslvr** [comment]
#47
